saleor/graphql/product/types/products.py

```diff
@@ -311,16 +318,25 @@ class Meta:
         interfaces = [relay.Node]
         model = models.ProductVariant
 
+    @staticmethod
+    @permission_required("stock.manage_stocks")
+    def resolve_stocks(root: models.ProductVariant, *_args):
+        stocks = stock_models.Stock.objects.select_related("warehouse").filter(
+            product_variant=root
+        )
+        return stocks
+
+    @staticmethod
+    @permission_required("stock.manage_stocks")
+    def resolve_stock_for_country(root: models.ProductVariant, info):
```

Comment:
It would be better if we will not assume that we have single stock for country (it can be changed in the future).  You have permission for both resolvers which means that customer will not be able to fetch current stock. Am I right?
Maybe we could have single resolver and  always return all avaiable stocks and storefront would decide how to handle the data?

Reply:
cc @szewczykmira  @maarcingebala

Reply:
Hmm, that seems ok for me, I'll just wait for @maarcingebala to confirm